src/code/z_scene.c

```diff
@@ -45,28 +45,37 @@ s32 Object_SpawnPersistent(ObjectContext* objectCtx, s16 objectId) {
     return objectCtx->numEntries - 1;
 }
 
+// PAL N64 versions reduce the size of object space by 4 KiB in order to make more room
+// for the Zelda arena, which can help prevent the "Hyrule Field glitch" where actors
+// do not have enough space to spawn.
```

Comment:
I think its okay to mention speedrunning names as a side note, but should not be in the main description. I would say
```
// PAL N64 versions reduce the size of object space by 4 KiB to give some space back to
// the Zelda arena, which can help prevent an issue where actors fail to spawn in specific areas
// (sometimes referred to as "Hyrule Field Glitch", even though it can happen in more places than Hyrule Field)
```